test/test-applications/integrations/aspnet/TestApplication.AspNet/TestApplication.AspNet.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just being extra careful here: what is the whitespace change here? Notice also the last line.

Reply:
I think that it has to do with the byte-order mark that Visual Studio manages.